app/assets/stylesheets/global.scss

```diff
@@ -3,6 +3,14 @@
   font-size: 13px;
   line-height: 45px;
   padding: 0 15px;
-  color: #fff;
+  color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.